App.js

```diff
@@ -4,38 +4,118 @@ function MainContainer() {
   return (
     <div>
       <colgroup className="MainContainer">
-        <SideBar />
+        <div>
+          <SideBar />
+          <UserButton />
+        </div>
```

Comment:
Should this be moved into another "_Col" component? Is `Sidebar` the right name here since the entire column is technically a sidebar?